### ISSUE TITLE
Definition for rule '@angular-eslint/template/i18n' was not found

### ISSUE THREAD

**Issue by muuvmuuv** [angular-eslint/angular-eslint#121]
Using `0.2.0-beta.1` and following the latest README update I have added that new *i18n* option but this gives me the below error.

```
  1:1  error  Definition for rule '@angular-eslint/template/i18n' was not found  @angular-eslint/template/i18n
```

<details>
<summary>.eslintrc.js</summary>

```js
const path = require('path')

module.exports = {
  root: true,
  plugins: ['tsdoc'],
  extends: [
    'eslint:recommended',
    'prettier/@typescript-eslint',
    'plugin:@typescript-eslint/recommended',
    'plugin:@angular-eslint/recommended',
    'plugin:prettier/recommended', // always last
  ],
  parserOptions: {
    tsconfigRootDir: path.resolve(__dirname, '..'),
    project: 'tsconfig.base.json',
    sourceType: 'module',
    ecmaVersion: 2020,
  },
  env: {
    es6: true,
    browser: true,
    webextensions: true,
    serviceworker: true,
  },
  ignorePatterns: ['.eslintrc.js'],
  rules: {
    'tsdoc/syntax': 'warn',
    // @typescript-eslint
    '@typescript-eslint/no-explicit-any': 'off',
    '@typescript-eslint/explicit-module-boundary-types': [
      'warn',
      {
        allowedNames: [
          'ngOnInit',
          'ngDoCheck',
          'ngOnChanges',
          'ngAfterContentInit',
          'ngAfterViewInit',
          'ngAfterViewChecked',
          'ngAfterContentChecked',
          'ngOnDestroy',
        ],
      },
    ],
    // @angular-eslint
    '@angular-eslint/no-input-rename': 'off',
  },
  overrides: [
    {
      files: ['*.component.html', '*.page.html'],
      parser: '@angular-eslint/template-parser',
      plugins: ['@angular-eslint/template'],
      rules: {
        '@angular-eslint/template/banana-in-a-box': 'error',
        '@angular-eslint/template/cyclomatic-complexity': 'error',
        '@angular-eslint/template/no-call-expression': 'error',
        '@angular-eslint/template/no-negated-async': 'error',
        '@angular-eslint/template/i18n': [
          'error',
          {
            checkId: false,
            checkText: true,
            checkAttributes: true,
            ignoreAttributes: ['field', 'identifier'],
          },
        ],
      },
    },
  ],
}
```

</details>

**Comment by JamesHenry** [comment]
Looks like there is an open PR for that rule: https://github.com/angular-eslint/angular-eslint/pull/96

It is not a part of the version you are using (or any version yet) because it hasn’t been merged yet
